staging/src/k8s.io/apiserver/pkg/util/flowcontrol/format/formatting.go

```diff
@@ -88,12 +88,12 @@ func ToJSON(val interface{}) string {
 
 // FmtPriorityLevelConfiguration returns a golang source expression
 // equivalent to the given value
-func FmtPriorityLevelConfiguration(pl *fcv1a1.PriorityLevelConfiguration) string {
+func FmtPriorityLevelConfiguration(pl *flowcontrol.PriorityLevelConfiguration) string {
 	if pl == nil {
 		return "nil"
 	}
 	var buf bytes.Buffer
-	buf.WriteString(fmt.Sprintf("&v1alpha1.PriorityLevelConfiguration{ObjectMeta: %#+v, Spec: ",
+	buf.WriteString(fmt.Sprintf("&flowcontrol.PriorityLevelConfiguration{ObjectMeta: %#+v, Spec: ",
```

Comment:
It might be wiser to have the version in the string.

Reply:
> It might be wiser to have the version in the string.

yeah showing the version in the alias can probably improve the readability, while it comes at the cost of maintainence. it took me a while to replace v1alpha1 -> v1beta1 everywhere in the codebase. if it's just flowcontrol, we can easily switch the versions by tweaking the package-importing lines ideally 🤔